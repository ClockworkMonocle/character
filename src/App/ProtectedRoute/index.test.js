import React from 'react';
import { shallow } from 'enzyme';
import { ProtectedRoute } from './index';

test('<ProtectedRoute /> renders', () => {
  expect(shallow(<ProtectedRoute user={{}} />)).toMatchSnapshot();
});
