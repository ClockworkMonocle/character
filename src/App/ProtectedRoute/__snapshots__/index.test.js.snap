// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProtectedRoute /> renders 1`] = `
<Route
  render={[Function]}
/>
`;
